tutorials/text/annotations.py

```diff
@@ -116,9 +158,9 @@
 # `~.Axes.text` takes a *bbox* keyword argument, which draws a box around the
 # text::
 #
-#     t = ax.text(
-#         0, 0, "Direction", ha="center", va="center", rotation=45, size=15,
-#         bbox=dict(boxstyle="rarrow,pad=0.3", fc="cyan", ec="b", lw=2))
+#     t = ax.text(0, 0, "Direction", ha="center", va="center", rotation=45,
+#         size=15, bbox=dict(boxstyle="rarrow,pad=0.3", fc="cyan", ec="b", lw=2)
+#         )
```

Comment:
When wrapping, if the first argument is next to the opening parenthesis, then the following lines should line up with the first argument. Alternatively, if the opening parenthesis is the end of the line, then the first argument is on the next line with an additional indent.

Additionally, since `bbox` is the important argument here, having it on a separate line seems better to me to call it out (ever so slightly).

So I would revert this change.

Reply:
so the convention that's used in like half the examples is that required args go on the first line and then everything else gets wrapped and that seems well suited to a tutorial so I tried to convert all the examples to that.